language-server/modules/langserver-core/src/test/resources/completion/module_var_context/source/source17.bal

```diff
@@ -0,0 +1,7 @@
+import ballerina/module1;
+
+configurable module1:
```

Comment:
Shouldn't we sort the members of `module1` based on subtypes of `anydata`?

Reply:
Yes. I have updated the sorting logic now. And added two test cases for both `ModulePartNodeContext` and `ModuleVarNodeContext`.

Reply:
Great! :+1: